src/js/components/Grommet/README.md

```diff
@@ -69,6 +69,17 @@ boolean
 min
 ```
 
+**options**
+
+Additional configurations that will be applied throughout the 
```

Comment:
@ShimiSun can you read this and see if it's more clear?

Reply:
I think it's better, but I still find it too technical and less descriptive.

For example:
> If options.layer.singleId === true

What is this prop `layer.singleId` and where do we control it (theme/prop)? 
Should we really use this code to explain the functionality or can we use words instead?
What is the purpose of this `options` prop and how can the user leverage it?